lib/olive_branch_patch.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# This will monkey patch olivebranch middleware https://github.com/vigetlabs/olive_branch/blob/master/lib/olive_branch/middleware.rb
+# so that when the VA inflection changes a key like something_va_something, the results will
+# have the old-inflection somethingVaSomething.
+
+module OliveBranchMiddlewareExtension
+  def call(env)
+    result = super(env)
+    _status, headers, response = result
+    # olive branch uses this private method to determine if a tranformation should happen https://github.com/vigetlabs/olive_branch/blob/master/lib/olive_branch/middleware.rb#L92
+    return result if send(:exclude_response?, env, headers)
+
+    if env['HTTP_X_KEY_INFLECTION'] =~ /camel/i
```

Comment:
you look for `HTTP_X_KEY_INFLECTION` here, but the specs use 'X-Key-Inflection'

Reply:
That's how it looks in the ENV in middleware